### PR TITLE
Adding project main app

### LINKED ISSUE #7: Create homepage for project demo 1
Create an HTML and CSS for project demo 1, including:
- Project name + logo
- Description of the project

After creating the website, a connection between Django and the website need to be created.

### PR DESCRIPTION
**Pull request by OfirMatasas**
Adding the main app, called "main", of the project, and the first page the client see when using the app is "homepage.html".

closes #7 
depends on #13

### CODE REVIEW

File: homepage/templates/homepage/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      <div class="u-layout">
+        <div class="u-layout-row">
+          <div
+            class="u-align-center-lg u-align-center-sm u-align-center-xl u-align-center-xs u-black u-container-style u-layout-cell u-left-cell u-size-30 u-layout-cell-1">
+            <div class="u-container-layout u-container-layout-1">
+              <h2 class="u-align-center-md u-text u-text-1">Welcome to</h2>
+              <h1 class="u-align-center-md u-text u-text-2">Hotails</h1>
+              <p class="u-align-center-md u-text u-text-grey-30 u-text-3">An online marketplace that connects people who
+                are looking for a daycare for their dogs,<br>&nbsp; &nbsp; &nbsp; &nbsp; with people want to keep other
+                people dogs for money.<br>
```
Comment (Yuval-Vino) [inline homepage/templates/homepage/index.html:41]: this sentence doesn't sound good to me (keep dogs for money :( ) 
maybe something like that Hotails main goal is to provide both for dogs owners and dog daycares managers,
an efficient, comfortable, and reliable system.

File: homepage/templates/homepage/index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+
+  <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-e5b7">
+    <div class="u-clearfix u-sheet u-sheet-1">
+      <p class="u-small-text u-text u-text-variant u-text-1">Project Demo 1 - Team 1</p>
+    </div>
+  </footer>
+</body>
+
+</html>
```
Comment (Yuval-Vino) [inline homepage/templates/homepage/index.html:146]: No newline at EOF error

File: static/Home.css [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
+
+@media (max-width: 575px) {
+  .u-section-2 .u-image-1 {
+    min-height: 391px;
+  }
+}
```
Comment (Yuval-Vino) [inline static/Home.css:341]: No newline at EOF error

File: static/Home.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    <p class="u-text">
+      <span>created with</span>
+    </p>
+    <a class="u-link" href="" target="_blank">
+      <span>Website Builder Software</span>
+    </a>.
+  </section>
+</body>
+
+</html>
```
Comment (Yuval-Vino) [inline static/Home.html:166]: No newline at EOF error

**Review by Yuval-Vino — COMMENTED** [review]
Ofir great job overall!! the site looks really good, ive added some small comments , tell me what you think about them!

File: .gitignore [hunk 5]
```diff
@@ -129,4 +129,7 @@ dmypy.json
 .pyre/
 
 # Vagrant
-.vagrant
\ No newline at end of file
+.vagrant
+
+# Pycharm .idea directory
+.idea
```
Comment (ErezCohenn) [inline .gitignore:135]: @jeniaSakirko already did it in another PR, take a look at #17

**Comment by Omeramsc** [comment]
Please fix the conflicts and squash the commits. 
(in order to fix a Pipfile.lock conflict, remove the existing one and re-create it)

File: homepage/templates/homepage/index.html (hunk 1 above)
Comment (OfirMatasas) [reply homepage/templates/homepage/index.html:41]: Sounds good
I'll edit it now

File: homepage/templates/homepage/index.html (hunk 2 above)
Comment (OfirMatasas) [reply homepage/templates/homepage/index.html:146]: Apparently VSCode deletes empty lines at the end of HTML files.
Fixing it now.

**Comment by OfirMatasas** [comment]
> Please fix the conflicts and squash the commits. (in order to fix a Pipfile.lock conflict, remove the existing one and re-create it)

Done.

**Comment by Omeramsc** [comment]
Please rebase with main and solve conflicts
that'll also make it easier to review, as right now there are too many commits. 
try to also squash the commits that are left.

**Comment by Omeramsc** [comment]
Homepage doesn't need a specific `application` of its own, it is only a `view` without any serious business logic behind it.
please refer to the django guides: https://docs.djangoproject.com/en/4.0/intro/tutorial01/#creating-the-polls-app

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Homepage doesn't need a specific application of its own, it is only a view without any serious business logic behind it.
please refer to the django guides: https://docs.djangoproject.com/en/4.0/intro/tutorial01/#creating-the-polls-app

Also, I didn't fully understand the PR description. what do you mean by ```Requesting adding the project web application for the repository.```

**Comment by OfirMatasas** [comment]
> Homepage doesn't need a specific application of its own, it is only a view without any serious business logic behind it. please refer to the django guides: https://docs.djangoproject.com/en/4.0/intro/tutorial01/#creating-the-polls-app

The idea of the application is to make the homepage the "main app" of the project, where the client can decide to join "Hotails" as a dog owner, or as a daycare owner.
For now, it's only informative web page, but right after this demo, it will become the actual app of this project.
If you still think I need to remove the app, I will.

**Comment by OfirMatasas** [comment]
> Also, I didn't fully understand the PR description. what do you mean by `Requesting adding the project web application for the repository.`

It was unnecessary for the description. Removed it.

**Comment by Omeramsc** [comment]
> > Homepage doesn't need a specific application of its own, it is only a view without any serious business logic behind it. please refer to the django guides: https://docs.djangoproject.com/en/4.0/intro/tutorial01/#creating-the-polls-app
> 
> The idea of the application is to make the homepage the "main app" of the project, where the client can decide to join "Hotails" as a dog owner, or as a daycare owner. For now, it's only informative web page, but right after this demo, it will become the actual app of this project. If you still think I need to remove the app, I will.

You do, because that is not what a "main app" is, it is what a "main page" is. the "main app" name should be the project's name. it can then include the homepage view and configs in it.

**Comment by Omeramsc** [comment]
I'll repeat our private discussion so everyone will be able to learn from it: 
Currently, the HTMLs are semi-auto-generated. While I'm sure Ofir knows how to use those tools, they'll be much harder to understand, produce and\or modify for the other team members. it is also harder and more complicated to review. 
I suggest moving to basic HTML-CSS while using the bootstrap library. 

in any way, for the first demo you need to have a simple, basic homepage up. we can make things fancy once we have more time. :)

**Review by ormergi — APPROVED** [review]
LGTM

**Comment by ormergi** [comment]
@OfirMatasas what about Omer's comment https://github.com/redhat-beyond/Hotails/pull/18#issuecomment-1064159581
Would you do it?

**Comment by OfirMatasas** [comment]
> @OfirMatasas what about Omer's comment https://github.com/redhat-beyond/Hotails/pull/18#issuecomment-1064159581
> 
> Would you do it?

As we spoke on Slack, I'll change it after the first demo, while we'll develop our project with other apps as needed, and I'll make sure that all team members understand how to work with HTML/CSS, and how can we make the project be good and "fancy" as one.

**Review by Omeramsc — APPROVED** [review]
LGTM
please open an issue for revamping the HTML.

**Pull request merged**